events/datadog.go

```diff
@@ -70,12 +68,10 @@ func (d *DataDog) NewEvent(title string, text string, fields map[string]string,
 	if err != nil {
 		return err
 	}
-	// TODO: fix?
-	// defer g.Conn.Close()
-	err = g.Event(title, text, fields, tags)
-	if err != nil {
-		return fmt.Errorf("Error reporting Godspeed event %s: %s", title, err)
-	}
+	defer g.Godspeed.Conn.Close()
+	g.W.Add(1)
```

Comment:
Do we care about waiting for these events to finish? There's not much point using the async interface if we're going to wait for calls to complete.